<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestMiscForm</class>
 <widget class="QWidget" name="TestMiscForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="pushButtonTestProjMat">
     <property name="text">
      <string>Test Projection Matrix</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonTesGlmColumRow">
     <property name="text">
      <string>Test Glm Column Row</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonTestTBN">
     <property name="text">
      <string>Test TBN</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
